src/technology-consulting/5-communication.md

```diff
@@ -9,23 +9,52 @@
 
 The following the communication tools that should be present on every project. The specifics on your project should be documented in the [learn-technology-consulting/transitioning-to-delivery#app-summary-document App Summary Document]
 
-- <input type="checkbox"/> Wiki - _GitHub Wiki_
-- <input type="checkbox"/> Screen Sharing - _Google Meet_ or _Slack Calls_
-- <input type="checkbox"/> Video Conference - _Google Meet_ or _Slack Calls_
-- <input type="checkbox"/> Chat - _Slack_
-- <input type="checkbox"/> Calendars - _Shared Google Calendars_ and a project/company calendar.
-- <input type="checkbox"/> Meetings - When meetings are held.
-
+- Wiki - _GitHub Wiki_
```

Comment:
For these, let's update this section to be the four types of scenarios we discussed re immediacy and complexity, and which tools are appropriate for which scenario.